webpack doesn't build: Can't import the named export 'version' (imported as 'version') from default-exporting module (only default export is available)

### Node Version

20.0.0

### NPM/Yarn/PNPM Version

NPM 10.0.0

### Strapi Version

5.10.3

### Operating System

Strapi Cloud

### Database

Strapi Cloud

### Javascript or Typescript

Javascript

### Reproduction URL

_No response_

### Bug Description

Suddenly my webpack doesn't build anymore. The issue is the following syntax:

`import { version } from '@strapi/admin/package.json';`


Which is part of @strapi/admin: https://github.com/strapi/strapi/blob/24165c9d5cbc756c03fe23bc491569c77973f5ac/packages/core/admin/admin/src/layouts/AuthenticatedLayout.tsx#L3

Introduced here https://github.com/strapi/strapi/commit/be068eb4403fd20e7c662dc05f5d18858a72dd3f by @jorrit 🙈 


Thats the second time webpack breaks in the last upates. Can you please test the webpack build when releasing new versions? I know its deprecated, but we are required to use it in our environment.

### Steps to Reproduce

strapi build --bundler=webpack

### Expected Behavior

strapi build --bundler=webpack should succeed

### Logs

```shell
strapi build --bundler=webpack

[WARN] [@strapi/strapi]: Using webpack as a bundler is deprecated. You should migrate to vite.
⠋ Building build context
[INFO] Including the following ENV variables as part of the JS bundle:
    - ADMIN_PATH
    - STRAPI_ADMIN_BACKEND_URL
    - STRAPI_TELEMETRY_DISABLED
✔ Building build context (50ms)
⠋ Building admin panel
⠸ Building admin panel[ERROR] assets by status 30.1 MiB [cached] 359 assets
Entrypoint main = runtime~main.4b40a84c.js main.75aed8e4.js
orphan modules 8.25 MiB [orphan] 2400 modules
runtime modules 13.8 KiB 16 modules
cacheable modules 26.8 MiB (javascript) 1.03 MiB (asset)
  javascript modules 26.7 MiB
    modules by path ./node_modules/ 26.4 MiB 1418 modules
    + 7 modules
  asset modules 71.2 KiB (javascript) 1.03 MiB (asset)
    modules by mime type image/svg+xml 60.1 KiB 76 modules
    modules by path ./ 1.03 MiB (asset) 2.46 KiB (javascript) 60 modules
    modules by mime type image/png 8.6 KiB
      data:image/png;base64,iVBORw0KGgoAAAAN.. 1.89 KiB [built] [code generated]
      + 4 modules
  json modules 62.4 KiB
    ./node_modules/markdown-it-emoji/lib/data/full.json 34 KiB [built] [code generated]
    ./node_modules/markdown-it/node_modules/entities/lib/maps/entities.json 28.4 KiB [built] [code generated]

ERROR in ./node_modules/@strapi/admin/dist/admin/chunks/AuthenticatedLayout-C9eZ12Ue.mjs 781:47-54
Can't import the named export 'version' (imported as 'version') from default-exporting module (only default export is available)
 @ ./node_modules/@strapi/admin/dist/admin/chunks/index-BM8-DG5K.mjs
 @ ./node_modules/@strapi/admin/dist/admin/index.mjs 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518
 @ ./node_modules/@strapi/strapi/dist/admin.mjs 1:0-57 2:0-43 2:0-43 13:9-20
 @ ./.strapi/client/app.js

ERROR in ./node_modules/@strapi/admin/dist/admin/chunks/AuthenticatedLayout-C9eZ12Ue.mjs 822:114-121
Can't import the named export 'version' (imported as 'version') from default-exporting module (only default export is available)
 @ ./node_modules/@strapi/admin/dist/admin/chunks/index-BM8-DG5K.mjs
 @ ./node_modules/@strapi/admin/dist/admin/index.mjs 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518
 @ ./node_modules/@strapi/strapi/dist/admin.mjs 1:0-57 2:0-43 2:0-43 13:9-20
 @ ./.strapi/client/app.js

ERROR in ./node_modules/@strapi/admin/dist/admin/chunks/AuthenticatedLayout-C9eZ12Ue.mjs 833:49-56
Can't import the named export 'version' (imported as 'version') from default-exporting module (only default export is available)
 @ ./node_modules/@strapi/admin/dist/admin/chunks/index-BM8-DG5K.mjs
 @ ./node_modules/@strapi/admin/dist/admin/index.mjs 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518 1:0-518
 @ ./node_modules/@strapi/strapi/dist/admin.mjs 1:0-57 2:0-43 2:0-43 13:9-20
 @ ./.strapi/client/app.js

webpack 5.96.1 compiled with 3 errors in 7543 ms
✖ Building admin panel
[ERROR]  There seems to be an unexpected error, try again with --debug for more information
```

### Code Snippets

Workaround: 

```js
// ./src/admin/webpack.config.js
"use strict";

/* eslint-disable no-unused-vars */
module.exports = (config, webpack) => {

  config.resolve.alias["@strapi/admin/package.json"] = false;

  return config;
};
```

### Media

_No response_

### Additional information

_No response_

### Confirmation Checklist

- [x] I have checked the existing [issues](https://github.com/strapi/strapi/issues) for duplicates.
- [x] I agree to follow this project's [Code of Conduct](https://github.com/strapi/strapi/blob/develop/CODE_OF_CONDUCT.md).

--- a/packages/core/admin/admin/src/layouts/AuthenticatedLayout.tsx
+++ b/packages/core/admin/admin/src/layouts/AuthenticatedLayout.tsx
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-import { version as strapiVersion } from '@strapi/admin/package.json';
+import packageInfo from '@strapi/admin/package.json';
 import { Box, Flex, SkipToContent } from '@strapi/design-system';
 import { DndProvider } from 'react-dnd';
 import { HTML5Backend } from 'react-dnd-html5-backend';
@@ -24,6 +24,8 @@ import { useMenu } from '../hooks/useMenu';
 import { useOnce } from '../hooks/useOnce';
 import { useInformationQuery } from '../services/admin';
 import { hashAdminUserEmail } from '../utils/users';
+
+const { version: strapiVersion } = packageInfo;
 
 const AdminLayout = () => {
   const setGuidedTourVisibility = useGuidedTour(

